CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Upcoming Release
+
+## Changed
+
+- [[#177](https://github.com/rust-vmm/linux-loader/pull/176)] Implemented loading of PVH module records.
```

Comment:
Mh, I think this should either be "Added" (if this was intentionally not implemented in the past) or "Fixed" (if it was an oversight).

Reply:
Thanks for the review! :) Updated the changeling entry (also expanded to make it less obscure)